src/lib/handleAuth/handleAuth.ts

```diff
@@ -1,9 +1,12 @@
 import {kindeAuthClient} from '$lib/KindeSDK.js';
 import {kindeConfiguration} from '$lib/index.js';
+import {sessionStorage} from '$lib/sessionStorage/sessionStorage.js';
 import {parseSearchParamsToObject} from '$lib/utils/index.js';
 import type {SessionManager} from '@kinde-oss/kinde-typescript-sdk';
 import {error, redirect, type RequestEvent} from '@sveltejs/kit';
 
+const KEY_POST_REDIRECT_URL = 'post-redirect-url';
```

Comment:
@prokawsar could we call this `post-login-redirect-url` to be consistent with the NextJS SDK - also enables us to do `post-logout-redirect-url` later if we want to

Reply:
Hello @DaveOrDead 
Thanks for your review. 
I updated with changes accordingly your comments. 
Also added relative and absolute url handler.